tests/functional/test_install.py

```diff
@@ -527,33 +527,17 @@ def test_install_package_with_target(script):
         str(result)
     )
 
-    # Test repeated call without --upgrade, no files should have changed
+    # Test repeated call, no files should have changed
     result = script.pip_install_local(
         '-t', target_dir, "simple==1.0", expect_stderr=True,
     )
     assert not Path('scratch') / 'target' / 'simple' in result.files_updated
 
-    # Test upgrade call, check that new version is installed
-    result = script.pip_install_local('--upgrade', '-t',
-                                      target_dir, "simple==2.0")
```

Comment:
For this test and the one below, you might want to keep this second invocation, but adapt it to test that `--target` doesn't overwrite existing packages. Or is that tested somewhere else I don't see?